api/java/src/test/java/com/qbot/answeringservice/service/CourseServiceTest.java

```diff
@@ -23,15 +23,6 @@ public class CourseServiceTest {
     @InjectMocks
     CourseService courseService;
 
-    @Test
-    public void testCreateCourse() {
-        Mockito.when(courseRepo.save(ArgumentMatchers.any(Course.class))).thenReturn(getTestCourses().get(0));
-
-        Course createdCourse = new Course("1", "MET", "CS", "633", null, null, null);
-        Course returnedCourse = courseService.createCourse(createdCourse);
-        Assertions.assertNotNull(returnedCourse);
```

Comment:
We can keep this test by replacing the `assertNotNull()` with a `verify(courseRepo, Mockito.times(1)).save(ArgumentMatchers.any(Course.class))`